Attacks/BaseAttack.gd

```diff
@@ -295,6 +296,12 @@ func checkDodged(_attacker, _receiver, _damageType, customDodgeMult = 1, minChan
 
 func getAttackSoloAnimation():
 	return ""
+	
+func getAttackHitReactAnimation(result):
```

Comment:
maybe you could pass the _receiver (and _attacker too, why not) into this function and then move all the if isBlocking: anim="blocking" checks into this function as well. Feels strange to only be able to override the hurt animation

Reply:
Fair point. Changes made.